changelog.md

```diff
@@ -1,4 +1,5 @@
 # Future
+- [ADDED] Expand string in `include` option into the association
```

Comment:
`include now supports string as an argument (rather than model/association), string will expand into an association matched literally from Model.associations`